<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewList</class>
 <widget class="QDialog" name="NewList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableView" name="tableViewAllMovies_new">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>15</y>
     <width>370</width>
     <height>500</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableView" name="tableViewNewList_new">
   <property name="geometry">
    <rect>
     <x>415</x>
     <y>15</y>
     <width>370</width>
     <height>500</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_new">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>540</y>
     <width>431</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonBack_new">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>540</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonAdd_new">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>550</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>ADD</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
